RSS Reader/RSS Reader/Model/RSSRTopic.m

```diff
@@ -64,4 +66,9 @@ - (NSString *)itemTitle {
     return self.title;
 }
 
+- (BOOL)isShowDetails {
+    return self.showDetails;
```

Comment:
Do you expect a positive result in case your discretion is an empty string or a string with an only a whitespace?

Reply:
Good point. Fixed it